manim/mobject/svg/text_mobject.py

```diff
@@ -324,71 +336,6 @@ def text2svg(self):
         return file_name
 
 
-# Following class is just a Little implementation of upcomming feautures. Ignore it for now.
-class TextWithBackground(Text):
```

Comment:
Is it safe to remove this? I think it has something to do with `Code` mobject. Does 'Code` mobject work fine after removing? If it does not cause problems, then it's fine if we remove it now and later add whatever upcoming feature it was going to implement.

Reply:
`TextWithBackground` was not used anywhere else within manim, and in the form it was implemented it just called `Text.__init__` and did nothing else.